### DISCUSSION TITLE
Unit cell transformation of hexagonal lattice vectors

### DISCUSSION THREAD

**Discussion by Bhuvaneswari-R432** [irreducible-representations/irrep#59] [Q&A]
Greetings,

I am working with a conventional hexagonal lattice, for which corresponding primitive rhombohedral lattice (FCC) exists. The command -searchcell works well with the primitive rhombohedral lattice vectors. However it could not find any shift that places the origin on an inversion center which leads to the expressions for the symmetries found in the table for conventional hexagonal lattice. Kindly help me in resolving this issue.

If I am to provide -refUC and -shiftUC separately, it involves floating integers, which again leads to RuntimeError. Please help me.

Regards,
Bhuvaneswari R

**Comment by stepan-tsirkin** [comment]
@Bhuvaneswari-R432  

In many cases such errors happen because no enough decimal digits are written in the POSCAR (or equivalent in other codes) , namely "0.3333" is not exactly "1/3" . please check if you have at least 8 (or better 10) accurate digits for the lattice vectors and atom positions. 

If that is not the case, please provide your complete POSCAR and the full exception traceback.
